lms/templates/video.html

```diff
@@ -141,5 +145,10 @@ <h3 class="hidden">${_('No playable video sources found.')}</h3>
             ${('<a href="%s" target="_blank">' + _('Download Handout') + '</a>') % handout}
         </li>
     % endif
+    % if settings.FEATURES.get('CREATIVE_COMMONS_LICENSING', False) and licensable and video_license:
+      <div class="xmodule_licensable license">
+        ${video_license.html}
```

Comment:
HTML should be written into this template, not generated in the `video_license.html` attribute.